src/trading/TradingEscapeHatch.sol

```diff
@@ -0,0 +1,94 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/trading/ITradingEscapeHatch.sol';
+import 'ROOT/trading/ICash.sol';
+import 'ROOT/trading/IOrders.sol';
+import 'ROOT/trading/IShareToken.sol';
+import 'ROOT/libraries/DelegationTarget.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+contract TradingEscapeHatch is DelegationTarget, ITradingEscapeHatch {
+    using SafeMathUint256 for uint256;
+
+    //      market             outcome  value
```

Comment:
style, I'm generally not a fan of vertical alignment of text like this.  I would personally prefer
```
// market => (outcome => value)
```
What do others think?